src/main/java/pl/edu/agh/firecell/engine/BasicEngine.java

```diff
@@ -20,7 +20,6 @@ public BasicEngine(SimulationConfig config, Storage storage, Algorithm algorithm
     @Override
     public void run() {
         engineThread.start();
-        logger.info("Engine run.");
```

Comment:
Why delete? If it is not necessary, change the level.

Reply:
I moved it to engine runnable, to log it if the engine actually runs.